topydo/Commands.py

```diff
@@ -32,7 +32,7 @@
     'depri': 'DepriCommand',
     'do': 'DoCommand',
     'edit': 'EditCommand',
-    'exit': 'ExitCommand', # used for the prompt
+    'exit': 'ExitCommand',  # used for the prompt
```

Comment:
It was intentional. This fixes PEP8 E261 ("at least two spaces before inline comment").